### ISSUE TITLE
Wireframes for Olin Circ Spring 2022

### ISSUE THREAD

**Issue by cappadona** [cul-it/signage#211]

**Comment by kevinkidwell** [comment]
I've created the first iteration of a wireframe for this signage. I based this on the work I did for the Mann signage back in 2019-2020. We had feedback from Cyrus in Disability Services on these mockups so I took that into account as well for this signage.

![Olin-Digital-Signage-v1 0](https://user-images.githubusercontent.com/29164102/154524101-c71f6ad8-fda0-40c2-8e91-2b950b201a04.jpg)

**Comment by cappadona** [comment]
Thanks @kevinkidwell. This looks good. My only comment is maybe we should alpha sort items under Laptops? Sorry, I realize I didn't have them alpha sorted in the Epic listing...I'll correct that now.

**Comment by cappadona** [comment]
Also...I'm trying to remember the WIP recommendations on datetime from the content style guide... @mhk33?

**Comment by mhk33** [comment]
@cappadona Chicago Style would be 7:00 p.m. Thanks for thinking of it!

**Comment by kevinkidwell** [comment]
That can make it a little harder with line length if we wanted to keep everything left-aligned in that right column because then it starts running into the text with the iPhone charger line. What do we think about having the availability column right-aligned?

![Olin-Digital-Signage-v1 0](https://user-images.githubusercontent.com/29164102/154550016-1eac170c-a757-470b-ba1b-94535dee0093.jpg)

**Comment by cappadona** [comment]
I can go either way, although I think I slightly prefer the left-aligned for scanning available counts.

Regardless of where we land, what do you think about dropping `Connector` from the iPhone label?

**Comment by kevinkidwell** [comment]
I also prefer the left-aligned and think dropping "Connector" makes a lot of sense, it's an extra word that we really don't need.

![Olin-Digital-Signage-v1 0](https://user-images.githubusercontent.com/29164102/154551844-2bb00004-1565-44c0-bdc5-e224a01c2d0f.jpg)

**Comment by mhk33** [comment]
This looks really good, by the way!

**Comment by cappadona** [comment]
Agreed. @kevinkidwell Do you want to share it with the stakeholders via our email thread?

**Comment by kevinkidwell** [comment]
Sure, I can respond to the thread with the mockup!
